frontend/wallet/src/Delete.re

```diff
@@ -1,7 +1,32 @@
 [@react.component]
-let make = () => {
-  <div>
+let make = (~closeModal, ~walletName) => {
+  <div onClick={e => ReactEvent.Synthetic.stopPropagation(e)}>
```

Comment:
why put this here instead of just putting it on the child click handlers?

Reply:
We want the event to stop propagating if you click anywhere in the modal content (aka anywhere in this div), not just the widgets within the div

```
----------------
|              |
|  |button|    |
|              |
----------------
```

Clicking on button shouldn't close the modal (unless it's the cancel button), but also clicking around the button shouldn't either.